packages/core/src/common/k8s-api/__tests__/api-manager.test.ts

```diff
@@ -43,7 +43,7 @@ describe("ApiManager", () => {
     di.override(directoryForKubeConfigsInjectable, () => "/some-kube-configs");
     di.override(storesAndApisCanBeCreatedInjectable, () => true);
 
-    const createCluster = di.inject(createClusterInjectable);
+    const createCluster = di.inject(createClusterInjectionToken);
```

Comment:
Reference equality is important here. Previously it worked because the implementation for `create-cluster` in `main` and `renderer` had same ID.